### PR TITLE
EVH Doc: Openshift EVH Support

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: docs/ako_evh.md [hunk 1]
```diff
@@ -1,50 +1,48 @@
 # Enhanced Virtual Hosting support in AKO
 
-This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. Currently, this feature is supported only for Kubernetes Ingress object.
+This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. This feature is supported for Kubernetes Ingress object and Openshift Route object.
```
Comment (akshayhavile) [inline docs/ako_evh.md:3]: @sudswasavi @monotosh-avi : should we specify here what different Openshift deployments we support in EVH explicitly?

File: docs/ako_evh.md [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
 
-With EVH (Enhanced Virtual Hosting) support in AVI, virtual hosting on virtual service can be enabled irrespective of SNI. Also, the SNI can only handle HTTPS (HTTP over SSL) traffic whereas EVH children can handle both HTTP and HTTPS traffic. Unlike SNI which switches only TLS (Transport Layer Security) connections based on one-to-one mapping of children to FQDN (Fully Qualified Domain Name), EVH maps one FQDN to many children based on the resource path requested.
+With EVH (Enhanced Virtual Hosting) support in AVI, virtual hosting on virtual service can be enabled irrespective of SNI. Also, the SNI can only handle HTTPS (HTTP over SSL) traffic whereas EVH children can handle both HTTP and HTTPS traffic. For each unique host in kubernetes/openshift cluster, unique EVH child will be created in secure, insecure deployments. L4 and Passthrough deployments will be similar to that of SNI way of deployment.
```
Comment (akshayhavile) [inline docs/ako_evh.md:9]: @sudswasavi @monotosh-avi : Is it ok to specify way of deployment (child per FQDN) here? In future, we will be changing model.

Comment (monotosh-avi) [reply docs/ako_evh.md:9]: I think it's okay to mention. Later we would modify the doc when it changes

File: docs/ako_evh.md (hunk 1 above)
Comment (monotosh-avi) [reply docs/ako_evh.md:3]: Do you mean with different CNIs for openshift or the version ?

Comment (akshayhavile) [reply docs/ako_evh.md:3]: @monotosh-avi : Different deployment modes like secure, insecure , edgetermination(allow/none/redirect) etc

**Comment by akshayhavile** [comment]
build ako

File: docs/ako_evh.md (hunk 1 above)
Comment (monotosh-avi) [reply docs/ako_evh.md:3]: I don't think we have to mention those, Ingress also has secure and insecure modes.

**Comment by monotosh-avi** [comment]
@akshayhavile master UTs would fail until https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/568 gets merged.

**Comment by akshayhavile** [comment]
build ako

**Pull request closed without merging**